src/test/java/fm/last/moji/impl/FileUploadOutputStreamTest.java

```diff
@@ -112,8 +112,6 @@ public void everyThingClosesEvenOnFail() throws IOException {
     verify(mockOutputStream).flush();
     verify(mockOutputStream).close();
     verify(mockHttpConnection).disconnect();
-    verify(mockTracker).createClose(KEY, DOMAIN, mockDestination, -1);
```

Comment:
This makes sense since we no longer call createClose() on the tracker if something fails. We're missing a test of the retry logic though - would be nice to see one where the first tracker createClose() fails but the second passes as well as one where both attempts fail and the lastException is thrown. In all cases the lock should be unlocked.